pkg/logs/launchers/integration/launcher.go

```diff
@@ -86,58 +118,138 @@ func (s *Launcher) run() {
 			for _, source := range sources {
 				// TODO: integrations should only be allowed to have one IntegrationType config.
 				if source.Config.Type == config.IntegrationType {
-					logFilePath, err := s.createFile(cfg.IntegrationID)
-					if err != nil {
-						ddLog.Warn("Failed to create integration log file: ", err)
-						continue
+					// This check avoids duplicating files that have already been created
+					// by scanInitialFiles
+					logFile, exists := s.integrationToFile[cfg.IntegrationID]
+
+					if !exists {
+						logFile, err = s.createFile(cfg.IntegrationID)
+						if err != nil {
+							ddLog.Warn("Failed to create integration log file:", err)
```

Comment:
Maybe this should be `Error` instead of `Warn`?

Reply:
Makes sense 👍

Reply:
I've also updated the log on line 114 to an `Error` since that would also prevent the agent from writing logs.